### DISCUSSION TITLE
Bypassing Age Verification Page

### DISCUSSION THREAD

**Discussion by kwr314** [GoogleChrome/lighthouse#14263] [Q&A]
Hello,

I'm running lambda tests on various websites that require the user to be over the age of 21 and because of this I do not think that Lighthouse is fully parsing the page. Every test I run shows screenshots of the age verification page and each page that I test receives very low performance scores. Any help with this issue would be appreciated.

Thanks 

![image](https://user-images.githubusercontent.com/110698299/183217766-d00266ad-a89e-47f4-8a65-7ffd76b3af94.png)

**Comment by adamraine** [comment]
You may want to look at https://github.com/GoogleChrome/lighthouse/blob/master/docs/authenticated-pages.md. You can use Puppeteer to script the age verification and then test Lighthouse.

You may also find https://github.com/GoogleChrome/lighthouse/blob/master/docs/user-flows.md helpful.

**Reply by kwr314** [reply]
Thanks
